### PR TITLE
Fixed two bugs

### PR DESCRIPTION
**Pull request by czardoz**
- Newman was not accepting Objects when given in the configuration
- Ensure that Newman sends a User-Agent header when fetching stuff

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -16,10 +17,10 @@ module.exports = {
         !callback && _.isFunction(options) && (callback = options, options = {});
         return (/^https?:\/\/.*/).test(location) ?
             // Load from URL
-            request.get({ url: location, json: true }, (err, response, body) => {
-                if (err) {
-                    return callback(err);
-                }
+            request.get({
+                url: location, json: true, headers: { 'User-Agent': 'Newman/' + version }
```
Comment (shamasis) [inline lib/util.js]: why compute this `Newman/' + version every time??!! Just compute once on top. :-/

Comment (shamasis) [inline lib/util.js]: How is this passing lint with multiple keys of object in one line???

File: lib/util.js [hunk 2]
```diff
@@ -16,10 +17,10 @@ module.exports = {
...
+            }, (err, response, body) => {
+                if (err) { return callback(err); }
```
Comment (shamasis) [inline lib/util.js]: What happened to the function end brace indentation here? Should it not be in new line?

Comment (czardoz) [inline lib/util.js]: It is on a different line.

**Pull request merged**